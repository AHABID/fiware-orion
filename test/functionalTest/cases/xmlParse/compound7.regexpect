7: +++++++++++++++++++++++++++++++++
HTTP/1.1 200 OK
Content-Length: 251
Content-Type: application/xml
Date: REGEX(.*)

<?xml version="1.0"?>
<updateContextResponse>
  <errorCode>
    <code>400</code>
    <reasonPhrase>Bad Request</reasonPhrase>
    <details>compound value error: bad tag-name of vector item: 'vecitem6' (should be 'vecitem')</details>
  </errorCode>
</updateContextResponse>
