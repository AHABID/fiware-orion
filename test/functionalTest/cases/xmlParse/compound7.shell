#   7. err: contextValue is a vector with five items + one item with a bad name
echo "7: +++++++++++++++++++++++++++++++++"
url='/ngsi10/updateContext'
payload='<?xml version="1.0"?>
<updateContextRequest>
  <contextElementList>
    <contextElement>
      <entityId type="Room" isPattern="false">
        <id>ConferenceRoom</id>
      </entityId>
      <contextAttributeList>
        <contextAttribute>
          <name>temperature</name>
          <type>degree</type>
          <contextValue type="vector">
            <vecitem>1</vecitem>
            <vecitem>2</vecitem>
            <vecitem>3</vecitem>
            <vecitem>4</vecitem>
            <vecitem>5</vecitem>
            <vecitem6>6</vecitem6>
          </contextValue>
        </contextAttribute>
      </contextAttributeList>
    </contextElement>
  </contextElementList>
  <updateAction>APPEND</updateAction>
</updateContextRequest>'
curlXml ${url} "${payload}"

