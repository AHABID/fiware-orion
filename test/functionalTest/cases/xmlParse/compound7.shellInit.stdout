{ "dropped" : "testharness", "ok" : 1 }
T@17:45:19  paConfigActions[727]: setting trace levels to '0-1'
E@17:45:19  exitFunc[1009]: unlink(/tmp/orion_9999.pid): No such file or directory
F@17:45:19  ***[1063]: Connected to mongo at localhost:testharness
V@17:45:19  restStart[569]: Starting HTTP daemon on IPv4 0.0.0.0 port 9999
V@17:45:19  restStart[611]: Starting HTTP daemon on IPv6 :: port 9999
2@17:45:20  xmlRequestGet[155]: Found xmlRequest of type 17, method 'POST' - index 12 (/updateContextRequest)
E@17:45:20  check[283]: bad tag-name of vector item: 'vecitem6' (should be 'vecitem')
W@17:45:20  compoundValueEnd[130]: ERROR: 'compound value error: bad tag-name of vector item: 'vecitem6' (should be 'vecitem')'
F@17:45:20  ***[981]: In sigHandler - caught signal 15
X@17:45:20  sigHandler[987]: Received signal 15
